spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarTemplateTests.java

```diff
@@ -134,9 +134,7 @@ void sendMessageWithSpecificSchemaTest() throws Exception {
 				pulsarTemplate.send(foo);
 
 				CompletableFuture<Message<Foo>> receiveMsgFuture = consumer.receiveAsync();
-				Message<Foo> msg = receiveMsgFuture.get(3, TimeUnit.SECONDS);
-
-				assertThat(msg.getValue()).isEqualTo(foo);
+				assertThat(receiveMsgFuture).isCompleted().isCompletedWithValueMatching(m -> m.getValue().equals(foo));
```

Comment:
Nice. A total **nit** is that I think `isCompletedWithValueMatching` implies the `isCompleted` and we only need the former.

Reply:
I will adjust during merge.